components/director/internal/graphql/scalars/annotations_test.go

```diff
@@ -0,0 +1,59 @@
+package scalars
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAnnotations_UnmarshalGQL(t *testing.T) {
```

Comment:
How about covering here case with multiple values, as we have also in Marshalling func.

Reply:
done